CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -6,7 +6,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.16 (TBC)
 
-<hr>
+- Compile binaries using [go 1.21.13](https://github.com/etcd-io/etcd/pull/18421).
```

Comment:
@henrybear327, I think we always keep the horizontal rule at the end of the release notes for that particular release. I think you're also missing adding the Dependencies header :smiley: 

```suggestion
### Dependencies
- Compile binaries using [go 1.21.13](https://github.com/etcd-io/etcd/pull/18421).

<hr>
```

Reply:
Thanks for catching this.

I reused the style from https://github.com/etcd-io/bbolt/blob/main/CHANGELOG/CHANGELOG-1.3.md#other when I did the update. Sorry for this...